tests/system/I18n/TimeTest.php

```diff
@@ -1139,4 +1139,25 @@ public function testSetTestNowWithFaLocale()
 
         Locale::setDefault($currentLocale);
     }
+
+    /**
+     * @dataProvider provideLocales
+     */
+    public function testToStringDoesNotDependOnLocale(string $locale)
```

Comment:
```suggestion
    public function testToStringDoesNotDependOnLocale(string $locale): void
```

Reply:
I did not add `void` because the code in this repository often does not have it.
No other test methods of this class have void.

Probably in the future, when we do a major version upgrade, we will add it with php-cs-fixer.

Reply:
Okay, Thanks👍

Reply:
I mean in test classes.
If we add a new public method, we should add parameter/return type.